src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -235,18 +235,45 @@ const myFade = {
 <header transition:animate={myFade}> ... </header>
 ```
 
-## Preventing client-side navigation
+## Router control
 
-Once you have enabled client-side routing on a page by adding the `<ViewTransitions />` component, every anchor on the page that links to another page on your site will be navigated via client-side routing. There are some cases where you might want to not navigate via CSR. One example is if the link is to any non-page, such as a PDF within your `public/` folder. Or an API route that produces an image.
+The `<ViewTransitions />` router handles navigation by listening to:
+
+- Clicks on `<a>` elements.
+- Backwards and forwards navigation events.
+
+The following sections discuss how you can further control when navigation occurs within the router.
+
+### Preventing client-side navigation
+
+There are some cases where you might want to not navigate via CSR. A transition will only occur when both the old and new page has the `<ViewTransitions />` component. There may be cases where you want a nice animation coming from one page, such as going from an article listing page `/articles` to an article detail page `/articles/emperor-penguins`, but you don't want an animation when coming from somewhere else on the site.
 
 In these cases, you can opt out of client-side routing on a per-link basis using the `data-astro-reload` attribute like so:
 
 ```html
-<a href="/quarterly-earnings.pdf" data-astro-reload>
+<a href="/articles/emperor-penguins" data-astro-reload>
 ```
 
 These links will be ignored by the router and a full page navigation will occur.
 
+### Trigger navigation
+
+There may be occasions when you need to trigger a navigation imperatively, such as via an event that occurs within JavaScript. The `astro:transitions/client` module provides a method for doing this. In any client script or client component you can use the module like so:
```

Comment:
```suggestion
You can also trigger client-side navigation via events not normally listened to by the `<ViewTransitions />` router using `navigate`. This method from the `astro:transitions/client` module is available in any client script or client component.

The following example shows using an option selected by a site visitor to trigger navigation: 
```
Improve the description as needed, but we do always try to describe what the code is doing so that "like this" isn't left up to the reader!